src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/CryptographicAttributeObjectCollection.cs

```diff
@@ -102,6 +102,22 @@ public int Count
             }
         }
 
+        public bool IsSynchronized
+        {
+            get
+            {
+                return ((ICollection)_list).IsSynchronized;
```

Comment:
Why did you change this from `return false;`?

Reply:
List should return `false` by default here. But I can change it to return false explicitly.

Reply:
Yes, please revert.  There's no value in paying for the extra call when it's always going to return false.

Reply:
addressed